src/common/util/firebase.ts

```diff
@@ -138,6 +138,19 @@ export function listenToRecentCards(callback: (data: any) => any): void {
     });
 }
 
+export function listenToUserRecentCards(callback: (data: any) => any, uid: string): void {
```

Comment:
I think `listenToRecentCards` should take an optional `uid` parameter rather than this code getting duplicated.

Reply:
Ah fair point